lib/ansible/modules/monitoring/zabbix/zabbix_screen.py

```diff
@@ -200,8 +197,6 @@ def get_host_ids_by_group_id(self, group_id, sort):
 
     # get screen
     def get_screen_id(self, screen_name):
-        if screen_name == "":
```

Comment:
Why remove this test? Is `""` a valid screen name? If yes, there should be a changelog fragment (bugfix since a valid screen name was not allowed by the module).

Reply:
`screen_name` is set to `required=True` in `argument_spec`. If an empty value is given `""` a module should fail implicitly rather than provide implicit message. In this case, this condition will never be met, but please correct me if I am mistaken in this behavior.

Reply:
`required=True` does not ensure that strings are not empty. It only makes sure that they aren't `None`. So it's better to keep this test.

Reply:
Reverted, thanks for the clarification! :)